.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# Contribution Guidelines
+
+Before filing a bug report or a feature request, be sure to read the contribution guidelines.
+
+## How to use GitHub
+We use GitHub exclusively for well-documented bugs, feature requests and code contributions. Communication is always done in English.
+
+To receive free support for Accessibility Checker we have the following channels:
+* [The support pages on our site](https://my.equalizedigital.com/support/)
+* [Support forums](https://wordpress.org/support/plugin/accessibility-checker/) on WordPress.org
+
+Thanks for your understanding.
+
+## Security issues
+Please do not report security issues here. Instead, email them to security at our domain so we can deal with them securely and quickly.
```

Comment:
@SteveJonesDev I need to created this email address. Do you want it to go to Zendesk? Or to you personally?

Reply:
Zendesk so it's tracked and addressed promptly.

Reply:
Should actually be a link to the Patchstack VDP 😅